habitat-lab/habitat/tasks/rearrange/utils.py

```diff
@@ -484,19 +489,6 @@ def get_robot_spawns(
         if physics_stability_steps == 0:
```

Comment:
Does this need to `sim.set_state(state)` before return ?

Reply:
This was done in the previous implementation, but I think we can remove that entirely. We can handle the robot state explicitly without saving/setting all the object states.